hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDistributedFileSystem.java

```diff
@@ -141,9 +141,9 @@ public class TestDistributedFileSystem {
   }
 
   private boolean dualPortTesting = false;
-  
+
```

Comment:
best to revert these, just to simplify merge/cherrypick

Reply:
will do this at the end